tests/sentry/workflow_engine/migration_helpers/test_migrate_alert_rule.py

```diff
@@ -0,0 +1,83 @@
+from sentry.incidents.grouptype import MetricAlertFire
+from sentry.snuba.models import QuerySubscription
+from sentry.testutils.cases import APITestCase
+from sentry.users.services.user.service import user_service
+from sentry.workflow_engine.migration_helpers.alert_rule import migrate_alert_rule
+from sentry.workflow_engine.models import (
+    AlertRuleDetector,
+    AlertRuleWorkflow,
+    DataSource,
+    DataSourceDetector,
+    Detector,
+    DetectorState,
+    DetectorWorkflow,
+    Workflow,
+    WorkflowDataConditionGroup,
+)
+from sentry.workflow_engine.types import DetectorPriorityLevel
+
+
+class AlertRuleMigrationHelpersTest(APITestCase):
+    def setUp(self):
+        self.metric_alert = self.create_alert_rule()
+        self.rpc_user = user_service.get_user(user_id=self.user.id)
+
+    def test_create_metric_alert(self):
```

Comment:
would it be possible to test the resulting models in workflow engine like `process_workflow_engine`? doesn't have to be in this PR